views/default/js/elgg/spinner.js

```diff
@@ -0,0 +1,66 @@
+define('elgg/spinner', function (require) {
+	var $ = require('jquery'),
+		elgg = require('elgg'),
+
+		num_running = 0,
+		$spinner;
+
+	var exports = {
```

Comment:
Add,remove,remove all? Smells like a collection api! Not sure what to do about that right now, just sort of interesting to me.